src/codegen/compile.ml

```diff
@@ -9145,13 +9180,18 @@ and compile_n_ary_pat env ae how pat =
     match pat.it with
     (* Nothing to match: Do not even put something on the stack *)
     | WildP -> None, G.nop
```

Comment:
```suggestion
    | WildP -> None, G.nop
```
How is `None` different from `Some (SR.UnboxedTuple 0`)? Just curious...

Reply:
None means please drop, and applies to values of any type.

UnboxedTuple 0 is an invalid stack rep for all types but `()`.

I think I tried to do as you say initially back then, but somehow that didn't work nicely.